torchvision/transforms/transforms.py

```diff
@@ -620,7 +620,7 @@ class RandomResizedCrop(object):
     """
 
     def __init__(self, size, scale=(0.08, 1.0), ratio=(3. / 4., 4. / 3.), interpolation=Image.BILINEAR):
-        if isinstance(size, tuple):
+        if isinstance(size, (tuple, list)):
```

Comment:
I agree with opening that up for more than just `tuple`s, but in that spirit wouldn't it be better to check for `collections.Sequence`?

Reply:
This is the most generic way of doing it, yes. But for the sake of simplicity (and to facilitate converting those transforms to torchscript in the near future) let's keep it as tuple and list for now.